packages/connect-examples/native-iOS-example/web/utils.js

```
@@ -0,0 +1,47 @@
+    localResolve = resolve;
+    localReject = reject;
+
+    if (typeof arg === 'function') {
+      try {
+        await arg();
+      } catch (error) {
+        reject(error);
+      }
+    }
+    if (typeof arg === 'string') id = arg;
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)